e2eTests/TestHomeButton

```diff
@@ -3,32 +3,32 @@
 <html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
 <head profile="http://selenium-ide.openqa.org/profiles/test-case">
 <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
-<link rel="selenium.base" href="http://0.0.0.0:3000/" />
-<title>test1</title>
+<link rel="selenium.base" href="http://0.0.0.0:3000/projects" />
+<title>TestHomeButton</title>
 </head>
 <body>
 <table cellpadding="1" cellspacing="1" border="1">
 <thead>
-<tr><td rowspan="1" colspan="3">test1</td></tr>
+<tr><td rowspan="1" colspan="3">TestHomeButton</td></tr>
 </thead><tbody>
 <tr>
 	<td>open</td>
-	<td>/</td>
+	<td>/projects</td>
 	<td></td>
 </tr>
 <tr>
-	<td>type</td>
-	<td>name=user</td>
-	<td>guinnrd</td>
+	<td>waitForElementPresent</td>
+	<td>css=span.c3PIvz</td>
```

Comment:
What are these numbers?

Reply:
or i guess just random characters

Reply:
The compiled element ID, at least as it is understood by selenium IDE. I changed the code and the ID stayed the same so I figured it was safe to use.

Reply:
yuck. is there any other way to reference it

Reply:
That is the only unique identifier present in the rendered HTML, there may be some non-html ways to find the element however, I will investigate.

Reply:
Were these files entirely automatically generated and executed by an IDE? If nobody ever has to see or edit the file I would say it is probably fine, but if this has to be human readable or editable I would imagine there must be a better way to reference it

Reply:
Yes, this will only be run in the IDE

Reply:
Is it generated from the IDE as well? Or do we have to look up the IDs to write the files? 

Can we verify that the number will never change in some of the Selenium documentation?

Reply:
The ID's are obtained by selenium from the HTML, this is what the TSX compiler generates I assume

Reply:
I have only tested changing the tsx files but I cannot be sure there is no way for it to change. A significant enough html could break any e2e test so it might be acceptable.
There is the option of adding a label or some other plain-text html element that is hidden but can be used as a testing tag.

Reply:
That is true. I guess I would suggest finding something in Selenium or TSX that suggests it should be static, or adding in a test tag / id and using that with all tests.